portal/academy/views_test.py

```diff
@@ -0,0 +1,22 @@
+import pytest
```

Comment:
Shouldn't this be placed in a `tests/` dir to follow the pattern of the rest of the project?

Reply:
Different habits 😅 I'll move it

Reply:
Done